pandas/core/indexes/interval.py

```diff
@@ -142,20 +142,24 @@ class IntervalIndex(IntervalMixin, Index):
 
     Parameters
     ----------
-    data : array-like (1-dimensional)
+    data : array-Like (1-dimensional)
```

Comment:
array-like is fine

Reply:
Also could you limit your changes to just the from_array method? Don't want conflicts with others.